zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -713,7 +713,7 @@ private void renameNote(NotebookSocket conn, HashSet<String> userAndRoles,
 
     NotebookAuthorization notebookAuthorization = notebook.getNotebookAuthorization();
     if (!notebookAuthorization.isOwner(noteId, userAndRoles)) {
-      permissionError(conn, "rename", fromMessage.principal,
+      permissionError(conn, "rename the note", fromMessage.principal,
```

Comment:
@tae-jun I didn't care about the other operations [as you said](https://github.com/apache/zeppelin/pull/1630#issuecomment-261773204). It was my bad. I just thought about the renaming.. How about just let this "rename the note" -> "rename" as before, and put `note` in [here](https://github.com/apache/zeppelin/pull/1630/commits/12b0bae387dca3f3898a274a315312441f682e95#diff-bcbdda87aa29314ed459e8212067e313R605)?